src/field_theory/minimal_polynomial.lean

```diff
@@ -190,18 +190,21 @@ lemma root {x : β} (hx : is_integral α x) {y : α}
 begin
   have ndeg_one : nat_degree (minimal_polynomial hx) = 1,
   { rw ← polynomial.degree_eq_iff_nat_degree_eq_of_pos (nat.zero_lt_one),
-    exact degree_eq_one_of_irreducible_of_root (irreducible hx) h },
+     exact degree_eq_one_of_irreducible_of_root (irreducible hx) h },
```

Comment:
```suggestion
    exact degree_eq_one_of_irreducible_of_root (irreducible hx) h },
```

Reply:
Could I suggest that reviewers commit small suggestions like this themselves? I don't think we need the overhead of leaving it to the author to commit, and switching labels back and forth.